src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/InputParser.cs

```diff
@@ -93,7 +93,7 @@ public IComputeRequest ParseInput(Query query)
 
                     if (!int.TryParse(versionQuery, null, out version))
                     {
-                        throw new ArgumentException("Could not determine requested GUID version");
+                        throw new ArgumentException("Could not determine requested GUID version. Supported versions are 1, 3, 4 and 5");
```

Comment:
Wondering if we should show the list of examples based on fuzzy search. Currently the query `uuidv` shows the error message and not the examples. Or at least decide if `uudiv` or `uuidv<unknown number>`.

Reply:
I'm changing `ArgumentException` into `FormatException` so it will still include the query in fuzzy search (same with SHA errors)